### PR TITLE
More verbose Alert Detail Fallback

### PR DESCRIPTION
**Pull request by SourceDoctor**
on Alert Details, return more data on fallback

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by PipoCanaja** [comment]
Could you attach 1 or 2 examples of this ?

**Comment by SourceDoctor** [comment]
sure
Screenshot of Alert Widget before:
image
Screenshot of Alert Widget after:
image

**Comment by PipoCanaja** [comment]
Hi,
Seems to have less impact than the previous one, for sure, even if is a bit noisy on some alerts that are not at all linked with ping ... Like Service Down for instance. 
A better solution should be found, but in the meantime, looks acceptable to me, at least with my set of rules and alerts in production and in tests. 
@librenms/reviewers If you are OK on this, feel free to approve & merge.

**Comment by SourceDoctor** [comment]
it's the fallback alert rule ;-)
if you remember weeks before it was extremly noisy with useless information.
Now with a less restrictive key filter more data is coming through with data, which might be usefull, also ping data ;-)

File: includes/html/functions.inc.php [hunk 1]
```diff
@@ -1008,7 +1008,7 @@ function alert_details($details)
                 if (in_array($k, ['device_id', 'sysObjectID', 'sysDescr', 'location_id'])) {
                     continue;
                 }
-                if (!empty($v) && (stristr($k, 'id') || stristr($k, 'desc') || stristr($k, 'msg')) && substr_count($k, '_') <= 1) {
```
Comment (murrant) [inline includes/html/functions.inc.php]: Code, looks ok to me, but could be simplified with str_i_contains()

```
str_i_contains(['id', 'desc', 'msg', 'last'], $k)
```

I'm not sure insensitive is needed either... so maybe could even use Str::contains()

Comment (SourceDoctor) [reply includes/html/functions.inc.php]: changed to it :-)

**Comment by SourceDoctor** [comment]
@PipoCanaja something still to do here?

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-61-release-changelog-february-2020/11145/1

**Pull request merged**